tests/Agent/IntegrationTests/IntegrationTests/InfiniteTracing/InfiniteTracingTests.cs

```diff
@@ -43,6 +43,7 @@ public InfiniteTracingTestsBase(TFixture fixture, ITestOutputHelper output) : ba
                     // Now send the command to make the 8T Span
                     _fixture.SendCommand("InfiniteTracingTester Make8TSpan");
                     // Now wait to see that the 8T spans were sent successfully
+                    _fixture.AgentLog.WaitForLogLinesCapturedIntCount(AgentLogBase.SpanStreamingSuccessfullySentLogLineRegex, TimeSpan.FromMinutes(1), ExpectedSentCount);
```

Comment:
Not for right now, but we should consider adding the ability to wait for multiple conditions, so we don't have to read and regex the whole log more than once.